modes/pdf/evil-collection-pdf.el

```diff
@@ -40,12 +42,16 @@
 (declare-function pdf-view-goto-page "pdf-view")
 (declare-function pdf-view-previous-line-or-previous-page "pdf-view")
 (declare-function pdf-view-next-line-or-next-page "pdf-view")
+(declare-function pdf-history-forward "pdf-history")
+(declare-function pdf-history-backward "pdf-history")
 
 (defvar pdf-view-mode-map)
+(defvar pdf-history-minor-mode-map)
 (defvar pdf-outline-buffer-mode-map)
 (defvar pdf-occur-buffer-mode-map)
 
 (defvar pdf-view-mode-map)
+(defvar pdf-history-minor-mode-map)
```

Comment:
Why there are duplicates? Cleanup.

It comes from this commit.

```diff
commit 323bb7d85848a6a142ae14f39c3a073ce6423e20
Author: James Nguyen <james@jojojames.com>
Date:   Sun Jan 24 00:56:41 2021 -0800

    Add scroll-lock (#358)

diff --git a/modes/pdf/evil-collection-pdf.el b/modes/pdf/evil-collection-pdf.el
--- /dev/null
+++ b/modes/pdf/evil-collection-pdf.el
@@ -0,0 +44,7 @@
+(defvar pdf-view-mode-map)
+(defvar pdf-outline-buffer-mode-map)
+(defvar pdf-occur-buffer-mode-map)
+
+(defvar pdf-view-mode-map)
+(defvar pdf-outline-buffer-mode-map)
+(defvar pdf-occur-buffer-mode-map)
```

And I can confirm we can remove one copy.

Reply:
Sorry, it's from 

```diff
commit a5303b07eedb8bb620c3489b8ee9b27744d49cb7
Author: Pierre Neidhardt <ambrevar@gmail.com>
Date:   Sun Feb 25 16:43:08 2018 +0100

    Silence compiler warnings

diff --git a/evil-collection-pdf.el b/evil-collection-pdf.el
--- a/evil-collection-pdf.el
+++ b/evil-collection-pdf.el
@@ -37,6 +37,10 @@
+(defvar pdf-view-mode-map)
+(defvar pdf-outline-buffer-mode-map)
+(defvar pdf-occur-buffer-mode-map)
+(declare-function evil-collection-inhibit-insert-state "evil-collection")
 
 (declare-function evil-collection-inhibit-insert-state "evil-collection")
 
 (defvar pdf-view-mode-map)
 (defvar pdf-outline-buffer-mode-map)
 (defvar pdf-occur-buffer-mode-map)

commit 5b2da79e7ea708224bb042cf4a45cc1d5397ed59
Author: James Nguyen <james@jojojames.com>
Date:   Sat Feb 24 17:14:05 2018 -0800

    Fix linting errors

diff --git a/evil-collection-pdf.el b/evil-collection-pdf.el
--- a/evil-collection-pdf.el
+++ b/evil-collection-pdf.el
@@ -37,1 +37,6 @@
 
+(declare-function evil-collection-inhibit-insert-state "evil-collection")
+
+(defvar pdf-view-mode-map)
+(defvar pdf-outline-buffer-mode-map)
+(defvar pdf-occur-buffer-mode-map)
```

The 2 commits do the same thing.